test/integration/node-specific/resource_tracking_script_builder.ts

```diff
@@ -0,0 +1,111 @@
+import { expect } from 'chai';
+import { fork } from 'child_process';
+import { on, once } from 'events';
+import { readFile, unlink, writeFile } from 'fs/promises';
```

Comment:
```suggestion
import { promises } from 'fs';
const { readFile, unlink, writeFile } = promises;
```
You should try this.

Reply:
thanks will do!